entity-service/src/test/resources/application-test.properties

```diff
@@ -1,3 +1,6 @@
 application.jsonld.aquac_context=https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/aquac/jsonld-contexts/aquac-compound.jsonld
 
 application.authentication.enabled = true
+
+spring.security.oauth2.resourceserver.jwt.issuer-uri = https://sso.eglobalmark.com/auth/realms/stellio
+spring.security.oauth2.resourceserver.jwt.jwk-set-uri = https://sso.eglobalmark.com/auth/realms/stellio/protocol/openid-connect/certs
```

Comment:
any reason to add them here? they are already defined with the same values in `application.properties`

Reply:
When changing the values in application.properties I ended up having the tests not working anymore.. Maybe that was a side effect on something else..

Reply:
Probably, the problem was elsewhere. What was the error encountered?

Reply:
```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'jwtDecoder' defined in class path resource [com/egm/stellio/entity/config/WebSecurityTestConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.security.oauth2.jwt.ReactiveJwtDecoder]: Factory method 'jwtDecoder' threw exception; nested exception is java.lang.IllegalArgumentException: Unable to resolve the Configuration with the provided Issuer of "https://lol.eglobalmark.com/auth/realms/stellio"
```
I changed the value to an host not existing and it does an error, that's likely why when I changed the variable in the ```application.properties``` of production code I got an error... 
This is just because we use the WebSecurityConfig of production code, which needs valid host.

I'll revert that.